apps/docs/src/app/docs/funding/page.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Funding
+nextjs:
+  metadata:
+    title: Funding
+    description: Blobscan funding
+---
+
+Blobscan was honored to be one of the [top finalists](https://twitter.com/ETHGlobal/status/1579249265557192704) at the [ETHBogota hackathon](https://bogota.ethglobal.com/) in 2022. Since then, numerous organizations and projects have generously supported and funded our vision.
+
+We deeply appreciate and extend our heartfelt gratitude to each one of them:
+
+- [Ethereum Foundation](https://ethereum.foundation/)
```

Comment:
We can also add a link to the blog post where available, year and amount of funding.